devnet/start.sh

```diff
@@ -16,6 +16,13 @@ while IFS= read -r line; do
         bootnodes="${bootnodes},$line"
     fi
 done <"$input"
+#check last line since it's not included in "read" command https://stackoverflow.com/questions/12916352/shell-script-read-missing-last-line
+if [ -z "${bootnodes}" ]
+then
+    bootnodes=$line
+else
+    bootnodes="${bootnodes},$line"
+fi
```

Comment:
Please use a tool to format shell files.

Reply:
hmm this looks same format as other if-then pattern in this script